src/solvers/flattening/boolbv_shift.cpp

```diff
@@ -30,9 +30,6 @@ bvt boolbvt::convert_shift(const binary_exprt &expr)
   if(width==0)
     return conversion_failed(expr);
 
-  if(expr.operands().size()!=2)
-    throw "shifting takes two operands";
-
```

Comment:
Is util/std_expr.h in sync with this?

Reply:
Yes, that's why the if(...) can go away.

Reply:
If it is something we assume then a DATA_INVARIANT would be best until we have the invariants in util/std_expr.h full enforced everywhere.  This is what I (and others) have been doing.

Reply:
Would still prefer a DATA_INVARIANT here until it is actually enforced by the code.